auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -155,7 +156,7 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
   // Local Properties
   private readonly businesses!: Business[]
   private readonly currentOrganization!: Organization
-  private readonly createNamedBusiness!: (filingBody: BusinessRequest) => any
+  private readonly createNamedBusiness!: (filingBody: any, nrNumber: string) => Promise<any>
```

Comment:
Why remove the typing?

Reply:
To support multiple type of filing. For now we have BusinessRequest and RegistrationRequest. Initially I thought of using `BusinessRequest | RegistrationRequest`. Then I changed it to any to support any future filing since it is only used to post a draft filing

Reply:
🤔  Perhaps we should modify the interfaces then to require the parts common to all filings, with an interchangeable subset for each unique filing?
I think changing the type to `any` is more or less removing typing, more so than supporting something.

Reply:
After offline discussion, we determined an acceptable path for the interfaces, will resolve!